addons/website/views/snippets/s_numbers_charts.xml

```diff
@@ -31,19 +31,20 @@
                         </div>
                     </div>
                 </div>
-                <div class="o_grid_item o_cc o_cc2 g-col-lg-5 g-height-12 col-lg-5 rounded" style="grid-area: 1 / 8 / 13 / 13; --grid-item-padding-y: 80px; --grid-item-padding-x: 40px; z-index: 4;">
+                <div class="o_grid_item o_cc o_cc2 g-col-lg-5 g-height-12 col-lg-5 rounded" style="grid-area: 1 / 8 / 13 / 13; --grid-item-padding-y: 64px; --grid-item-padding-x: 40px; z-index: 4;">
                     <div class="s_chart" data-type="doughnut" data-legend-position="none" data-tooltip-display="false" data-stacked="false" data-border-width="2" data-data='{"labels":["First","Second"],"datasets":[{"label":"One","data":["25","75"],"backgroundColor":["o-color-3","o-color-2"],"borderColor":["",""]}]}' data-snippet="s_chart" data-name="Chart">
+                        <p><br/></p>
                         <canvas style="box-sizing: border-box;" width="456" height="228"/>
                     </div>
                     <!-- Placeholder chart, to be displayed in the preview modal -->
-                    <svg class="s_dialog_preview d-block mx-auto" width="450" height="230" viewBox="0 0 100 110" xmlns="http://www.w3.org/2000/svg">
+                    <svg class="s_dialog_preview d-block mt-3 mx-auto" width="450" height="230" viewBox="0 0 100 110" xmlns="http://www.w3.org/2000/svg">
                         <circle cx="50" cy="50" r="40" fill="transparent" stroke="transparent" stroke-width="25"/>
                         <circle cx="50" cy="55" r="40" fill="transparent" stroke="var(--o-color-2)" stroke-width="25"  stroke-dasharray="251.2" stroke-dashoffset="62.8"/>
                     </svg>
                     <!-- End of placeholder -->
                     <p><br/></p>
                     <p class="display-1" style="text-align: center;">75%</p>
-                    <p class="o_small text-600" style="text-align: center;">75% of clients use the service for over a decade consistently.<br/>This showcases remarkable loyalty and satisfaction with the quality provided.</p>
+                    <p class="o_small text-600" style="text-align: center;">75% of clients have been using the service for over a decade consistently.</p>
```

Comment:
Translation update in stable: normally not ok even for fixing typo, although I am often ok with it. Some would absolutely not agree with me. Indeed, this means that french users creating a new DB will see this in English for weeks, which is not good.

I would be ok but... in this case you are also removing a `<br/>` element... which thus fallbacks to being an XML *change* on top of a translation change... not really safe.

I'd say: could you keep the `<br/>`? Then I would be ok with the term change. Although, your PR would then be missing the `.pot` update. You have to go to Settings -> Translation -> Export to update the `website.pot` file of the repo.